klaviyo/api_helper.py

```diff
@@ -247,13 +247,13 @@ def _handle_response(self, response):
             raise KlaviyoRateLimitException(status_code, response)
         elif status_code in (500, 503, ):
             raise KlaviyoServerError(status_code, response)
-        elif status_code != 200:
+        elif status_code != 200 and status_code != 202:
             raise KlaviyoAPIException(status_code, response)
 
-        return self.__handle_200_response(response, status_code)
+        return self.__handle_20X_response(response, status_code)
 
-    def __handle_200_response(self, response, status_code):
-        """Determines how to handle a 200 http response.
+    def __handle_20X_response(self, response, status_code):
```

Comment:
can we use sunder instead of dunder now!
```suggestion
    def _handle_successful_response(self, response, status_code):
```